components/dashboard.jsx

```diff
@@ -41,9 +41,27 @@ export default () => {
                     backgroundColor: '#fff',
                     cursor: 'pointer',
                   }}
-                />
+                ></div>
```

Comment:
So, if I have it as...

```
line 34: <div className="card-action" />
                <PopupButton
                  url={eventType.scheduling_url}
                  rootElement={document.getElementById('root')}
                  text="View Availbility"
                  styles={{
                    borderWidth: 0,
                    backgroundColor: '#fff',
                    cursor: 'pointer',
                  }}
                />
line 45: </div>
```

The UI is different, not in a good way. Haha.

Reply:
Was @maxt95 talking about the div being self-closing or PopupButton? The div wraps PopupButton, so it can't be self-closing, but PopupButton could be, and it looks like it was before this change. Now it looks like PopupButton isn't closed at all, but I guess React is okay with that?